htdocs/api/v0.0.1/Login.php

```diff
@@ -67,17 +67,17 @@ function handlePOST()
             $this->header("HTTP/1.1 400 Bad Request");
             $this->JSON = array("error" => "Missing username or password");
             return;
-
         }
         $user     = $this->RequestData['username'];
         $password = $this->RequestData['password'];
 
         $login = $this->getLoginAuthenticator();
 
         if ($login->passwordAuthenticate($user, $password, false)) {
-            $this->JSON = array(
-                           "token" => $this->getEncodedToken($user),
-                          );
+            $token = $this->getEncodedToken($user);
+            if (!empty($token)) {
+                $this->JSON = array("token" => $token);
+            }
```

Comment:
This was leftover from a previous change where I was returning an empty string instead of using a redirect in `getEncodedToken`. I will revert this portion to the original code